pennylane/templates/embeddings.py

```diff
@@ -68,6 +68,7 @@
 ^^^^^^^^^^^^
 """
 #pylint: disable-msg=too-many-branches,too-many-arguments,protected-access
+import numpy as np
```

Comment:
The numpy import should go below the collections import, as collections is a standard library module, but NumPy is a third party module.

Reply:
oh, I did not know. Thanks!

Reply:
There is a cool Python module called `isort` that can sort imports automatically for you. Simply
```
pip install isort
```
and then you can run
```
isort file.py
```